app/fissile.go

```diff
@@ -824,6 +828,68 @@ func (f *Fissile) GenerateKube(rolesManifestPath, outputDir, repository, registr
 		}
 	}
 
+	if createHelmChart {
```

Comment:
I kind of agree, but then the whole function (or even file) should be refactored. Do you want this done as part of this task?